indico/modules/categories/blueprint.py

```diff
@@ -16,9 +16,12 @@
 
 from __future__ import unicode_literals
 
-from indico.modules.categories.controllers import RHCategoryStatistics
+from indico.modules.categories.controllers import (RHCategoryStatistics, RHCategoryMoveContents,
+                                                   RHCategoryInfo)
 from indico.web.flask.wrappers import IndicoBlueprint
 
 _bp = IndicoBlueprint('categories', __name__, template_folder='templates', url_prefix='/category')
 
 _bp.add_url_rule('/<categId>/statistics', 'category-statistics', RHCategoryStatistics)
+_bp.add_url_rule('/<categId>/manage/move/', 'move-contents', RHCategoryMoveContents)
+_bp.add_url_rule('/<categId>/info', 'info', RHCategoryInfo, methods=('POST',))
```

Comment:
but without the `<categId>` the `url_for` won't build the URL passing the `Category` object..